webpack.config.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  entry     : 'main.js', resolve : {
+  root : 'src/UI',
+  alias : {
+      'templates'               : '/templates',
+      'backbone'                : 'Shims/backbone',
+      'moment'                  : 'JsLibraries/moment',
+      'filesize'                : 'JsLibraries/filesize',
+      'handlebars'              : 'Shared/Shims/handlebars',
+      'handlebars.helpers'      : 'JsLibraries/handlebars.helpers',
+      'bootstrap'               : 'JsLibraries/bootstrap',
+      'backbone.deepmodel'      : 'JsLibraries/backbone.deep.model',
+      'backbone.pageable'       : 'JsLibraries/backbone.pageable',
+      'backbone-pageable'       : 'JsLibraries/backbone.pageable',
+      'backbone-pageable'       : 'JsLibraries/backbone.pageable',
```

Comment:
I can understand the . and -, but - is list twice.
I understand it's straight from the require file, doesn't mean the duplicate shouldn't be fixed.